### ISSUE TITLE
BLU AF Hat Quest "Transformations" Won't Give CS (Imperial Whitegate)

### ISSUE THREAD

**Issue by Momo027** [EdenServer/community#1289]
The 2nd step of the quest seems to be bugged.  The "Imperial Whitegate" wont give any CS.  It doesn't respond at all, no dialogue text or anything.

I've double checked that I have the quest flagged under ToAU Current Quests and Waoud even recited the exact phrase described on the FFXI wiki: "Your fate lies beyond the gate of nobility".
I've tried everything from zoning, re-logging, waiting for the next game day, but the door still won't give any CS.

**Comment by E5una** [comment]
I'm in the same state that Momo was in in March. Would really appreciate help progressing, thanks!

Note: Maybe not the same state... Waod says this for me:

![image](https://user-images.githubusercontent.com/13668758/64913917-edc6ec80-d716-11e9-8180-a7cc1c454934.png)

**Comment by E5una** [comment]
TLDR: The Nashmau Route listed here https://ffxiclopedia.fandom.com/wiki/Transformations doesn't seem to work on Eden, but the Nyzul Isle one works fine.

Update to my last post. I think it was user error, but user error that stemmed from another bug :). To get the rest of BLU AF, you have to start transformations, so this is something I had done some time ago. I made the assumption I had already done the appropriate cutscenes in the past, and followed the wiki through the Nashmau Route to get my cutscenes since I didn't have Nyzul Isle staging point. I was unable to get the CS. I went back thinking I must have missed a prior step, and that's when I made my original post, thinking one of these pieces (like the door) must not be working.

I talked to another BLU the other day that told me they did it from the Nyzul Isle side, so I tried that and everything worked fine.
